Docs/improve seo (#256)
iotaledger/stronghold.rs: documentation/docs/specs/overview.md

---

> description: This section contains a description of the scope of the project, and working documents that detail the engineering and requirements specifications.

Reviewer: ```suggestion
description: This section describes the scope of the project and includes working documents on engineering and requirements specifications.
```
